File: java&html/src/main/java/pt/unl/fct/di/apdc/firstwebapp/util/AlertPostData.java
package pt.unl.fct.di.apdc.firstwebapp.util;

public class AlertPostData {

    private String creator;
    private String location;
    private String description;
    private long timestamp;

    public AlertPostData(){}

    public AlertPostData(String creator, String location, String description, long timestamp){
        this.creator = creator;
        this.location = location;
        this.description = description;
        this.timestamp = timestamp;
    }

    public String getCreator() {
        return creator;
    }

    public String getLocation() {
        return location;
    }

    public String getDescription() {
        return description;
    }

    public long getTimestamp() {
        return timestamp;
    }
}

File: flutter/fctconnect/lib/models/AlertPostData.dart
class AlertPostData {
  final String creator;
  final String location;
  final String description;
  final int timestamp;

  AlertPostData({
    required this.creator,
    required this.location,
    required this.description,
    required this.timestamp,
  });

  factory AlertPostData.fromJson(Map<String, dynamic> json) {
    return AlertPostData(
      creator: json['creator'],
      location: json['location'],
      description: json['description'],
      timestamp: json['timestamp'],
    );
  }
}